scripts/check-expo-dependencies.js

```diff
@@ -0,0 +1,39 @@
+module.exports = async ({ context, github, fetch, prTitle }) => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const expoVersion = require('../package.json').dependencies.expo
+  const expoVersionSanitized = expoVersion.replaceAll('~', '')
+
+  const majorExpoVersion = `${expoVersionSanitized.substring(
+    0,
+    expoVersionSanitized.indexOf('.')
+  )}.0.0`
+
+  const expoData = await fetch('https://api.expo.dev/v2/versions/latest')
+  const expoDataOutput = await expoData.json()
+
+  const { relatedPackages } = expoDataOutput.data.sdkVersions[majorExpoVersion]
+
+  const shouldClosePR = [
+    ...Object.keys(relatedPackages),
+    'react',
```

Comment:
this doesn't look very safe. I mean surely there are dependencies with the "react" string occurring in their name that we don't want to block. I'd try to be more specific about the dependency names

Reply:
Now I'm using `dependabot-metadata` which returns the list of dependency names being updated to match the related packages to avoid the problem you noticed above.